poster/poster.tex

```diff
@@ -0,0 +1,367 @@
+\captionsetup{compatibility=false}
+\usepackage{pgfplots, graphicx, amsmath,amsthm,amssymb,mathtools,   natbib,tikz} 
+\usepackage{array}
+\usepackage{booktabs}
+\usetikzlibrary{arrows,snakes,backgrounds,shapes, decorations.markings}
+\usetikzlibrary{matrix}
+\usetikzlibrary{chains}
+\usetikzlibrary{positioning}
+\usetikzlibrary{arrows.meta, shadows, arrows, positioning}
+
+
+%-----------------------------------------------------------
+% Define the column width and poster size
+% To set effective sepwid, onecolwid and twocolwid values, first choose how many columns you want and how much separation you want between columns
+% The separation I chose is 0.024 and I want 4 columns
+% Then set onecolwid to be (1-(4+1)*0.024)/4 = 0.22
+% Set twocolwid to be 2*onecolwid + sepwid = 0.464
+%-----------------------------------------------------------
+
+\newlength{\sepwid}
+\newlength{\onecolwid}
+\newlength{\twocolwid}
+\newlength{\threecolwid}
+\setlength{\paperwidth}{48in}
+\setlength{\paperheight}{36in}
+%four columns settings
+%\setlength{\sepwid}{0.024\paperwidth}
+%\setlength{\onecolwid}{0.22\paperwidth}
+%\setlength{\twocolwid}{0.464\paperwidth}
+%\setlength{\threecolwid}{0.708\paperwidth}
+%\setlength{\topmargin}{-0.5in}
+
+%three columns settings
+\setlength{\sepwid}{0.024\paperwidth}
+\setlength{\onecolwid}{0.3013\paperwidth}
+\setlength{\twocolwid}{0.626\paperwidth}
+\setlength{\threecolwid}{0.626\paperwidth}
+\setlength{\topmargin}{-0.5in}
+
+%major theme colors
+\usetheme{confposter}
+\usepackage{exscale}
+\usepackage{verbatim}
+%-----------------------------------------------------------
+% The next part fixes a problem with figure numbering. Thanks Nishan!
+% When including a figure in your poster, be sure that the commands are typed in the following order:
+% \begin{figure}
+% \includegraphics[...]{...}
+% \caption{...}
+% \end{figure}
+% That is, put the \caption after the \includegraphics
+%-----------------------------------------------------------
+
+\usecaptiontemplate{
+\small
+\structure{\insertcaptionname~\insertcaptionnumber:}
+\insertcaption}
+
+%-----------------------------------------------------------
+% Define colours (see beamerthemeconfposter.sty to change these colour definitions)
+%-----------------------------------------------------------
+
+\setbeamercolor{block title}{fg=SFUred,bg=white}
+\setbeamercolor{block body}{fg=black,bg=white}
+\setbeamercolor{block alerted title}{fg=white,bg=SFUred}
+\setbeamercolor{block alerted body}{fg=black,bg=SFUred!10}
+
+%make sure figures are numbered
+\setbeamertemplate{caption}[numbered]
+
+%define theorem environments
+\setbeamertemplate{theorems}[numbered]
+\newtheorem{proposition}[theorem]{Proposition}
+
+%-----------------------------------------------------------
+% Name and authors of poster/paper/research
+%-----------------------------------------------------------
+
+\title{A Neural Network Approach to Classifying Banana Ripeness }
```

Comment:
Do we want to mention Deep Learning or convolution neural networks in the title? I'm not sure, I actually like it as you wrote it.